aws/terraform/storage.rego

```diff
@@ -1210,6 +1309,45 @@ s3_website_metadata := {
     "Resource Help URL": "https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket.html"
 }
 
+
+#
+# PR-AWS-TRF-V4-S3-013
+#
+
+default s3_website_v4 = null
```

Comment:
@harshilswan 
how do we know these testcase are only getting applied to v4 of terraform aws provider?
should we check the provider version as well?